crates/solver/src/main.rs

```diff
@@ -184,6 +185,32 @@ struct Arguments {
     )]
     transaction_strategy: Vec<TransactionStrategyArg>,
 
+    /// Which access list estimators to use. Multiple estimators are used in sequence if a previous one
+    /// fails. Individual estimators might support different networks.
+    /// `Tenderly`: supports every network.
+    /// `Web3`: supports every network.
+    #[clap(
+        long,
+        env,
+        default_values = &["Web3", "Tenderly"],
```

Comment:
nit: Can we make the default an empty list. My rationale is:
- I _think_ if you have default values for `use_value_delimiter` parameters, there is actually no way to configure it to use an empty list. At least this was the case a few months ago.
- Including `Tenderly` in the default requires an access key - this is annoying to configure for local testing
- Including `Web3` requires a node with `createAccessList` support which is not always the case for local testing or for all networks (does this work on GChain and Rinkeby for example?)

That being said, I do think that the value for our Mainnet solver pod should be what you have here, just maybe it makes more sense to set that in our Kubernetes deployment configuration instead of hardcoded defaults in the binary.

Reply:
Yes, makes sense. Removed default.

> Including Web3 requires a node with createAccessList support which is not always the case for local testing or for all networks (does this work on GChain and Rinkeby for example?)

It does require a node with `createAccessList` support, however the code should not panic in case we define `Web3` and `createAccessList` is not supported on that node, because we agreed to have the access list functionality as an additional bonus, and to fallback to no access list if one does not exist for any reason.
Both client node (if it implements `createAccessList`) and tenderly support GChain and Rinkeby.